python/pyspark/worker.py

```diff
@@ -19,18 +19,27 @@
 Worker that receives input from Piped RDD.
 """
 from __future__ import print_function
+
 import os
+import socket
 import sys
 import time
-import socket
 import traceback
 
+from pyspark import shuffle
 from pyspark.accumulators import _accumulatorRegistry
-from pyspark.broadcast import Broadcast, _broadcastRegistry
+from pyspark.broadcast import Broadcast
+from pyspark.broadcast import _broadcastRegistry
 from pyspark.files import SparkFiles
-from pyspark.serializers import write_with_length, write_int, read_long, \
-    write_long, read_int, SpecialLengths, UTF8Deserializer, PickleSerializer, BatchedSerializer
-from pyspark import shuffle
+from pyspark.serializers import BatchedSerializer
+from pyspark.serializers import PickleSerializer
+from pyspark.serializers import SpecialLengths
+from pyspark.serializers import UTF8Deserializer
+from pyspark.serializers import read_int
+from pyspark.serializers import read_long
+from pyspark.serializers import write_int
+from pyspark.serializers import write_long
+from pyspark.serializers import write_with_length
 
```

Comment:
I have the habit to rearrange import statements. It is more readable and maintainable, and ease merges. I can move this to external pr.